src/x86/avx.rs

```diff
@@ -1,6 +1,8 @@
 #[cfg(test)]
 use stdsimd_test::assert_instr;
 
+use std::mem;
```

Comment:
I try to group imports in three groups: std imports, external crate imports and then finally internal imports. So I'd write it like this:

```rust
use std::mem;

#[cfg(test)]
use stdsimd_test::assert_instr;

use v256::*;
```